src/pfcdemo.c

```diff
@@ -64,6 +65,34 @@ static const int         NUMCOUNTS;
 int main(int argc, char* argv[]){
 	int i;
 	
+	int verbose = 0, dump = 0, option;
```

Comment:
Make `pfcdemo` a bit richer, with a couple of command line options, for dumping and enabling verbose mode.